Dockerfile

```diff
@@ -1,38 +1,40 @@
 FROM osgeo/gdal:ubuntu-small-3.4.1 as builder
 LABEL maintainer="info@camptocamp.com"
 
-RUN apt update && \
-    apt upgrade --assume-yes && \
-    LC_ALL=C DEBIAN_FRONTEND=noninteractive apt install -y bison flex python-lxml libfribidi-dev swig \
+RUN apt-get update && \
+    apt-get upgrade --assume-yes && \
+    LC_ALL=C DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes --no-install-recommends bison \
+    flex python-lxml libfribidi-dev swig \
     cmake librsvg2-dev colordiff libpq-dev libpng-dev libjpeg-dev libgif-dev libgeos-dev libgd-dev \
     libfreetype6-dev libfcgi-dev libcurl4-gnutls-dev libcairo2-dev libxml2-dev \
     libxslt1-dev python-dev php-dev libexempi-dev lcov lftp ninja-build git curl \
     clang libprotobuf-c-dev protobuf-c-compiler libharfbuzz-dev libcairo2-dev librsvg2-dev && \
-    apt clean && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN ln -s /usr/local/lib/libproj.so.* /usr/local/lib/libproj.so
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    ln -s /usr/local/lib/libproj.so.* /usr/local/lib/libproj.so
```

Comment:
I never noticed this linking, is this still useful?

Reply:
Yes, removing => 
```
#16 3.302 CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
#16 3.302 Please set them or make sure they are set and tested correctly in the CMake files:
#16 3.302 PROJ_LIBRARY (ADVANCED)
#16 3.302     linked by target "mapserver" in directory /src
#16 3.302 
#16 3.302 -- Configuring incomplete, errors occurred!
```